qdrant-landing/content/articles/late-interaction-models.md

```diff
@@ -0,0 +1,292 @@
+---
+title: "Every* model might be a late interaction model if you give it a chance"
+short_description: "Standard dense embedding models are surprisingly good in late interaction scenarios."
+description: "Standard dense embedding models are surprisingly good in late interaction scenarios."
+preview_dir: /articles_data/late-interaction-models/preview
+social_preview_image: /articles_data/late-interaction-models/social-preview.png
+weight: -160
+author: Kacper Łukawski
+author_link: https://kacperlukawski.com
+date: 2024-08-05T00:00:00.000Z
+---
+
+\* At least every Open Source model, because you need to access its internals.
+
+Qdrant 1.10 introduced support for multi-vector representations, and late interaction, are the most prominent example of 
+such models. In a nutshell, both documents and queries are represented by multiple vectors, and finding the most 
+relevant documents requires calculating a score based on similarity between the pairs of query and document embeddings.
```

Comment:
```suggestion
relevant documents require calculating a score based on the similarity between the query and document embeddings pairs.
```

Reply:
Thanks! Actually, "requires" refers to "finding" in that case, so I believe that was the correct form. However, I'm taking all the other changes :smiley: